third_party/move/move-compiler-v2/tests/testsuite.rs

```diff
@@ -580,6 +580,32 @@ const TEST_CONFIGS: Lazy<BTreeMap<&str, TestConfig>> = Lazy::new(|| {
             stop_after: StopAfter::SecondAstPipeline,
             ..config().exp(Experiment::MESSAGE_FORMAT_JSON)
         },
+        TestConfig {
+            name: "cmp-rewrite-lang-v2.1",
+            runner: |p| run_test(p, get_config_by_name("cmp-rewrite-lang-v2.1")),
+            include: vec!["/cmp-rewrite/"],
+            exp_suffix: Some("v2.1.exp"),
+            dump_ast: DumpLevel::EndStage,
+            dump_bytecode: DumpLevel::EndStage,
+            ..config().lang(LanguageVersion::V2_1)
+        },
+        TestConfig {
+            name: "cmp-rewrite-lang-v2.2",
+            runner: |p| {
+                let mut test_config = get_config_by_name("cmp-rewrite-lang-v2.2");
+                let mut options = test_config.options.clone();
+                options.dependencies.push(path_from_crate_root(
+                    "../../../aptos-move/framework/move-stdlib/sources",
```

Comment:
We do not want any third party stuff to have references to outside third party, which this does. We may have to move tests out of third party.